### PR TITLE
con-button styles have bad defaults

### PR DESCRIPTION
**Pull request by ryanmparrish**
The default `con-button` looks invisible without a `outline`, or `blue` style being set on the class list. 

* Updated the `con-button` default styles to match `outline` and removed any redundant styles
* Add to the `typography.css` file to support `aria-disabled="true"` styles and fixed sort order/css-linting

Resolves: [MWPW-130320](https://jira.corp.adobe.com/browse/MWPW-130320)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/rparrish/buttons
- After: https://rparrish-button-default--milo--adobecom.hlx.page/drafts/rparrish/buttons

### CODE REVIEW

File: libs/styles/typography.css [hunk 1]
```diff
@@ -154,14 +161,33 @@
   width: 100%;
 }
 
+.dark .con-button {
+  border-color: #fff;
+  color: #fff;
```
Comment (overmyheadandbody) [inline libs/styles/typography.css:166]: Could `--color-white` not be used here as well?

**Review by overmyheadandbody — APPROVED** [review]

File: libs/styles/typography.css (hunk 1 above)
Comment (ryanmparrish) [reply libs/styles/typography.css]: yea, might as well. I updated em. Thanks

**Review by Sartxi — APPROVED** [review]

**Review by yesil — APPROVED** [review]

**Comment by yesil** [comment]
@ryanmparrish unrelated to this PR but the italic button doesn't look italic:

image

**Review by rgclayton — APPROVED** [review]
Thanks @ryanmparrish for the quick turn around.

Looks got be solving for https://rparrish-button-default--milo--adobecom.hlx.page/drafts/ilyas/merch

Checked on consumer sites too. Not seeing any issues.

**Comment by rgclayton** [comment]
> @ryanmparrish unrelated to this PR but the italic button doesn't look italic:
> 
> image

@yesil The italic variant is the outline. You won't see italic text in the button. Bold created the sold blue button.

**Pull request merged**